<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DifficultMode</class>
 <widget class="QMainWindow" name="DifficultMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>789</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DifficultMode</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(50, 168, 82)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(94, 189, 120);
font-family: Arial;
padding: 4px;
color: white;</string>
    </property>
    <property name="text">
     <string>Player Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(94, 189, 120);
font-family: Arial;
padding: 4px;
color: white;</string>
    </property>
    <property name="text">
     <string>Start/Restart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DealButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(94, 189, 120);
font-family: Arial;
padding: 4px;
color: white;</string>
    </property>
    <property name="text">
     <string>Deal Cards</string>
    </property>
   </widget>
   <widget class="QLabel" name="hearts2label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>cards/2_of_clubs.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="diamonds2label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>cards/2_of_clubs.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="clubs2label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>cards/2_of_clubs.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="spades2label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>cards/2_of_clubs.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="heart3label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>cards/2_of_clubs.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="diamonds3label">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>50</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>cards/2_of_clubs.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="clubs3label">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>50</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>cards/2_of_clubs.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="spades3label">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>50</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>cards/2_of_clubs.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="heart3label_500">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="heart3label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="heart3label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>440</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="heart3label_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="heart3label_5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="heart3label_6">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>170</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="heart3label_7">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>170</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="heart3label_8">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>170</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="heart3label_9">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>170</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="heart3label_10">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>170</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="heart3label_11">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="heart3label_12">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="heart3label_13">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>300</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="heart3label_14">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>300</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="heart3label_15">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>300</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="heart3label_16">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>300</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="heart3label_17">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>300</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="heart3label_18">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>440</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="heart3label_19">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>440</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="heart3label_20">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>440</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="heart3label_21">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>440</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="heart3label_22">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>440</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="heart3label_23">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>440</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="heart3label_24">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>440</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dealtcardlabel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>580</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 159, 36);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>cards/4_of_hearts.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="gamestate">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>650</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Game ongoing...</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>889</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
